node/tests/SharedTests.ts

```diff
@@ -19,6 +20,13 @@ import {
     parseInfoResponse,
     GeoUnit,
 } from "../";
+import {
+    BitmapIndexType,
+    ConditionalChange,
+    GeospatialData,
+    UpdateOptions,
+} from "../build-ts/src/Commands";
+import { SingleNodeRoute } from "../build-ts/src/GlideClusterClient";
```

Comment:
Should we update simple route - make it as enum too?

Reply:
I think this one is a bit more complicated. I think it has to be in a separate PR for testing purposes as well. Thoughts?

Reply:
I agree, we can do it in another PR